<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibTool</class>
 <widget class="QMainWindow" name="CalibTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalibTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="accessibleName">
    <string extracomment="dd"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="calib">
       <attribute name="title">
        <string>calibrating</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>501</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="8" column="3">
          <widget class="QLabel" name="label_c2_x_v">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QLabel" name="label_c1_y_v">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="label_c1_x_v">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>CAM3</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="label_x_c2">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>CAM2</string>
           </property>
          </widget>
         </item>
         <item row="4" column="6">
          <widget class="QLabel" name="label_s_c1">
           <property name="text">
            <string>SIZE:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="6">
          <widget class="QLabel" name="label_s_c2">
           <property name="text">
            <string>SIZE:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QLabel" name="label_y_c1">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>CAM1</string>
           </property>
          </widget>
         </item>
         <item row="8" column="4">
          <widget class="QLabel" name="label_y_c2">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Actual cameras values:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_x_c1">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QLabel" name="label_x_c3">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="4">
          <widget class="QLabel" name="label_y_c3">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="6">
          <widget class="QLabel" name="label_s_c3">
           <property name="text">
            <string>SIZE:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="3">
          <widget class="QLabel" name="label_c3_x_v">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="8" column="5">
          <widget class="QLabel" name="label_c2_y_v">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="10" column="5">
          <widget class="QLabel" name="label_c3_y_v">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="7">
          <widget class="QLabel" name="label_c1_s_v">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="8" column="7">
          <widget class="QLabel" name="label_c2_s_v">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="10" column="7">
          <widget class="QLabel" name="label_c3_s_v">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="8">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="8">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="8">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>280</y>
          <width>556</width>
          <height>207</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="6" column="3">
          <widget class="QSpinBox" name="spinBox_2"/>
         </item>
         <item row="6" column="6">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_y_r">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSpinBox" name="spinBox"/>
         </item>
         <item row="2" column="0" colspan="6">
          <widget class="Line" name="line_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_x_r">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_35">
           <property name="text">
            <string>REAL COORDS</string>
           </property>
          </widget>
         </item>
         <item row="6" column="5">
          <widget class="QPushButton" name="saveVector">
           <property name="text">
            <string>save vector</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" rowspan="2" colspan="6">
          <widget class="Line" name="line_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="7">
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item row="11" column="6">
          <widget class="QPushButton" name="saveToFileButton">
           <property name="text">
            <string>Save as ...</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_datas_nr">
           <property name="text">
            <string>Nr of datasets :</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLabel" name="label_datas_nr_v">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="train">
       <attribute name="title">
        <string>training</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>821</width>
          <height>621</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>num neurons (hidden)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="spinBox_5"/>
         </item>
         <item row="3" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_4"/>
         </item>
         <item row="6" column="2">
          <widget class="QSpinBox" name="spinBox_7"/>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_3"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>fun activ steepness (output)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="spinBox_6"/>
         </item>
         <item row="4" column="2">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_4"/>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>check integrity</string>
           </property>
          </widget>
         </item>
         <item row="12" column="2">
          <widget class="QPushButton" name="button_start_training">
           <property name="text">
            <string>start training</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0" colspan="3">
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Integrity :</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_train_output">
           <property name="text">
            <string>Ouput:</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QPushButton" name="loadFromFileButton">
           <property name="text">
            <string>Load training set from file...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>use step </string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>fun activ steepness (hidden)</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>step : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>num layers</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Network generator:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" rowspan="3" colspan="3">
          <widget class="QTextEdit" name="text_edit_output"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>New</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSave_as">
    <property name="title">
     <string>Save as..</string>
    </property>
   </widget>
   <addaction name="menuNew"/>
   <addaction name="menuOpen"/>
   <addaction name="menuSave"/>
   <addaction name="menuSave_as"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
